spec/system/documents/index_spec.rb

```diff
@@ -54,6 +54,49 @@
 
       expect(current_url).to include("/rails/active_storage/")
     end
+
+    context "when there is more than one document" do
+      let(:file2) do
+        Rack::Test::UploadedFile.new(
+          Rails.root.join("spec/fixtures/images/proposed-roofplan.png"),
```

Comment:
I've added another png file because all the other test files in `/spec/fixtures/images` are PDFs. When I tried to write a test that opened any of these PDFs in a new window it kept timing out on github actions - I'm guessing due to the file size?